.flake8

```diff
@@ -1,5 +1,16 @@
 [flake8]
-ignore = E203, E266, E501, W503, F403, F401, C901
-max-line-length = 79
-max-complexity = 18
-select = B,C,E,F,W,T4,B9
\ No newline at end of file
+ignore = 
```

Comment:
Making this file more readable. Also adding back F401 unused inputs because it's a reasonable warning to have.